platform/oasis/transaction.go

```diff
@@ -0,0 +1,54 @@
+package oasis
+
+import (
+	"github.com/trustwallet/golibs/coin"
+	"github.com/trustwallet/golibs/types"
+)
+
+func (p *Platform) GetTxsByAddress(address string) (types.TxPage, error) {
```

Comment:
please rebase master, `GetTxsByAddress` should return `types.Txs` now
```suggestion
func (p *Platform) GetTxsByAddress(address string) (types.Txs, error) {
```